src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGenerator.cs

```diff
@@ -392,7 +393,13 @@ private OpenApiSchema CreateObjectSchema(DataContract dataContract, SchemaReposi
                     ? GenerateSchemaForMember(dataProperty.MemberType, schemaRepository, dataProperty.MemberInfo, dataProperty)
                     : GenerateSchemaForType(dataProperty.MemberType, schemaRepository);
 
-                if ((dataProperty.IsRequired || customAttributes.OfType<RequiredAttribute>().Any())
+                if ((
+                    dataProperty.IsRequired
+                    || customAttributes.OfType<RequiredAttribute>().Any()
```

Comment:
Are there any other places we use `[Required]` that also need updating to account for `[RequiredMember]`?

Reply:
I've updated the usage in `GenerateSchemaForMember`.